electrum/wallet.py

```diff
@@ -1747,19 +1747,26 @@ def fee_estimator(size: Union[int, float, Decimal]) -> int:
                     lower_bound = lower_bound if not is_local else 0
                     return int(max(lower_bound, original_fee_estimator(size)))
                 txi = base_tx.inputs()
-                txo = list(filter(lambda o: not self.is_change(o.address), base_tx.outputs()))
+                txo = list(filter(lambda o: not self.is_change(o.address), base_tx.outputs())) + list(outputs)
+                output_dict = {}
+                for output in txo:
+                    if output.address in output_dict:
+                        output_dict[output.address].value += output.value
+                    else:
+                        output_dict[output.address] = copy.copy(output)
+                txo = list(output_dict.values())
```

Comment:
Note: In general re the whole idea of merging same-address outputs in batch-RBF, the same question arises in `PartialTransaction.join_with_other_psbt`.

Reply:
https://github.com/spesmilo/electrum/pull/8474/commits/f5a58675f131140de60c38a43194bdd5d6fdde1d

Reply:
@SomberNight I added this also for PSBTs: https://github.com/spesmilo/electrum/pull/8474/commits/16375d8c2b50dd978fb89e0da0debfa49582e41a. Is that ok?